bfsscp/bfsscp.go

```diff
@@ -72,10 +74,22 @@ type Config struct {
 	Prefix string
 	// A custom temp dir.
 	TempDir string
+	// Whether Prefix Paths are absolute or relative
+	Absolute bool
 }
 
 func (c *Config) norm() error {
-	c.Prefix = strings.TrimPrefix(c.Prefix, "/")
+	// By default prefixes should start in the user's dir
+	if !c.Absolute {
+		c.Prefix = strings.TrimPrefix(c.Prefix, "/")
```

Comment:
@ajn probably, make absolute a default and do smth like this?

```go
if strings.HasPrefix(c.Prefix, "/~/") || strings.HasPrefix(c.Prefix, "/./") {
  ...
}
```

Reply:
I think we had this problem in some other adapter or on the ruby side, how did we solve it there? I like Max' solution too

Reply:
Gone with the above